### DISCUSSION TITLE
Integrate with Jenkins

### DISCUSSION THREAD

**Discussion by flextra19** [community/community#30103] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, how can i integrate Git with Jenkins?

**Comment by bojanterzic529** [comment, marked as answer]
Step 1. Click on the manage jenkins button on your jenkins dashboard.
Step 2. Click on manage jenkins plugin.
Step 3: In the Plugins Page

Select the GIT Plugin
Click on Install without restart. The plugin will take a few moments to finish downloading depending on your internet connection, and will be installed automatically.
You can also select the option Download now and Install after restart  In which plugin is installed after restart
You will be shown a “No updates available” message if you already have the Git plugin installed.
Step 4: Once the plugins have been installed, go to Manage Jenkins on your Jenkins dashboard. You will see your plugins listed among the rest.

I will be glad if this help you

**Reply by flextra19** [reply]
It's working well, solved the problem.  Thanks a lot for your help, @bojanterzic529.
